src/main/java/hudson/plugins/scm/koji/KojiChangeLogSet.java

```diff
@@ -83,4 +78,67 @@ public User getAuthor() {
 
     }
 
+    public static class HyperlinkStringContainer {
```

Comment:
Do I understand it correctly, that this class has 2 purposes, to hold hyperlink or string? It feels just bad.